### DISCUSSION TITLE
How is the smooth transition implemented?

### DISCUSSION THREAD

**Discussion by Jseeca** [MonitorControl/MonitorControl#1133] [Q&A]
Monitors use eeprom memory to store the brightness/contrast settings, as a result using DDC control directly uses this memory whem charging settings. Eeprom have a finite writes. So I was wondering with the smooth transition how is it implemented? 

When you change from one interval to another is it incrementally adjusting the actual brightness/contrast of the monitor (Hence excessive wear) or it adjusts in set intervals and the smoothness is achieved by software. 

I currently have it off, but if it works the latter way, then turning the smooth transition increases wear by a factor of 6 since each interval is 6 increments (for my monitor, yours will vary). Want to prolong my monitor, so helpful to know. Thanks!

**Comment by waydabber** [comment]
Yes, this is a theoretical possibility. Most displays however give you a 1% increment/decrement value change in their OSD and write the values as soon as there is a change, nor did I ever heard of a display actually having issues due to excessive DDC writes yet - but it is certainly possible that the flash/eprom storage of a display fails, so if you want to be super careful or use the display for many years/decades to come, the best you can do is to turn off smooth transition and directly hit the proper point on the slider so brightness is directly set to the right level in one step.
